docs/guide/replacement-guides/rimraf.md

```diff
@@ -0,0 +1,43 @@
+# Replacements for `rimraf`
+
+## Node.js
+
+If you are using [`rimraf`](https://github.com/isaacs/rimraf) to remove files and directories, modern Node.js provides native alternatives. Since Node v14.14.0, [`fs.rm`](https://nodejs.org/api/fs.html#fspromisesrmpath-options) supports recursive deletion and works as a direct replacement.
```

Comment:
we don't need this first bit since anyone reaching this doc is definitely using rimraf. it can be simplified to something like:

> Node.js v14.14.0 and above provide a native alternative [...]